app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import datetime
+import time
+
+
+def create_a_file_every_second():
+    while True:
+        now = str(datetime.datetime.now())
+        file_name = f"app-{now[11:13]}_{now[15:17]}_{now[18:20]}.log"
```

Comment:
You can use [`strftime`](https://www.programiz.com/python-programming/datetime/strftime) for `datetime.datetime.now()` formatting and there no need to convert it to `str`

Reply:
done, tnx )